virtual_ship/instruments/ship_underwater_st.py

```diff
@@ -42,26 +49,31 @@ def simulate_ship_underwater_st(
     particleset = ParticleSet.from_list(
         fieldset=fieldset,
         pclass=_ShipSTParticle,
-        lon=0.0,  # initial lat/lon are irrelevant and will be overruled later.
+        lon=0.0,  # initial lat/lon are irrelevant and will be overruled later
         lat=0.0,
         depth=depth,
         time=0,  # same for time
     )
 
     # define output file for the simulation
-    out_file = particleset.ParticleFile(
-        name=out_file_name,
-    )
+    # the default outputdt is good(infinite), as we want to just want to write at the end of every call to 'execute'
```

Comment:
Still, I would then explicitly set `outputdt=np.inf`, as the default may(?) change at some point in parcels?

Reply:
fair enough. adjusted